### PR TITLE
Add git.repository_url for Source Code Integration

### PR DESCRIPTION
**Pull request by jybp**
### What does this PR do?

Add the `git.repository_url` tag automatically like the `git.commit.sha` tag for #source-code-integration
The original PR that added the commit tag is https://github.com/DataDog/serverless-plugin-datadog/pull/202

### Motivation

The tag will be required by #source-code-integration

### Testing Guidelines

I attempted `yarn build` but got a lot of errors that seem unrelated. I'm not sure why 🤔 

```
% yarn build
yarn run v1.22.11
$ ./scripts/check_layers_json.sh
$ tsc
src/env.ts:9:21 - error TS2307: Cannot find module 'serverless/classes/Service' or its corresponding type declarations.

9 import Service from "serverless/classes/Service";
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~

src/env.ts:137:13 - error TS2339: Property 'environment' does not exist on type 'ExtendedFunctionDefinition'.

137     handler.environment ??= {};
                ~~~~~~~~~~~
...
src/wrapper.ts:41:20 - error TS2339: Property 'package' does not exist on type 'ExtendedFunctionDefinition'.

41       func.handler.package.include = [];
                      ~~~~~~~


Found 83 errors.

error Command failed with exit code 2.
```

Yarn version:
```
yarn --version
1.22.11
```

### Types of changes

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change
- [ ] Misc (docs, refactoring, dependency upgrade, etc.)

### Check all that apply

- [ ] This PR's description is comprehensive
- [ ] This PR contains breaking changes that are documented in the description
- [ ] This PR introduces new APIs or parameters that are documented and unlikely to change in the foreseeable future
- [ ] This PR impacts documentation, and it has been updated (or a ticket has been logged)
- [ ] This PR's changes are covered by the automated tests
- [ ] This PR collects user input/sensitive content into Datadog

### CODE REVIEW

**Comment by IvanTopolcic** [comment]
Is this change live? Will source code integration still work without this?

**Comment by IvanTopolcic** [comment]
Is this change live? Will source code integration still work without this?

**Comment by jybp** [comment]
> Is this change live? Will source code integration still work without this?

Moved this discussion to Slack.

**Review by IvanTopolcic — APPROVED** [review]

File: src/index.ts [hunk 1]
```diff
@@ -399,13 +399,13 @@ module.exports = class ServerlessPlugin {
...
-    const gitCommitHash = await sourceCodeIntegration.uploadGitMetadata();
+    const info = await sourceCodeIntegration.uploadGitMetadata();
 
-    this.serverless.cli.log(`Adding GitHub integration with git commit hash ${gitCommitHash}`);
+    this.serverless.cli.log(`Adding GitHub integration with git commit hash ${info.hash} and remote ${info.remote}`);
 
     handlers.forEach(({ handler }) => {
       handler.environment ??= {};
-      handler.environment[ddTagsEnvVar] = "git.commit.sha:" + gitCommitHash;
+      handler.environment[ddTagsEnvVar] = "git.commit.sha:" + info.hash + " git.repository_url:" + info.remote;
```
[the remark this answers is not in the document]
Comment (IvanTopolcic) [reply src/index.ts:408]: We're going to hold off on merging this for right now while we look into which of our services use a comma-separated DD_TAGS vs space-separated.

**Review by IvanTopolcic — APPROVED** [review]

**Comment by tamirshaul** [comment]
Hey all, first of all thanks for that change!
However it causes us an issue with bitbucket.

When deploying through bitbucket the tags now look like this:
```
DD_TAGS=git.commit.sha:8609584c43a4583bf30ea531841601ef1fcab22f,git.repository_url:http://bitbucket.org/xxxx/repo-name
```

This causes this error in CloudWatch:
```
[ERROR]	2022-06-23T09:13:42.374Z		
Malformed tag string with tags not separated by comma or space 'git.commit.sha:8609584c43a4583bf30ea531841601ef1fcab22f,git.repository_url:http://bitbucket.org/xxxx/repo-name.
```

This happens because the git url isn't quoted and contains ':'. it causes datadog to think it is a beginning of a new tag and that the tags aren't separated by comma.

Let me know if you need me to open a separate bug for this.
I'll be happy to fix it myself but I don't think I'll get to it this week.

Thanks in advance!

**Pull request merged**